src/devices_models/devices/electric_loads.jl

```diff
@@ -135,7 +135,7 @@ function activepower_constraints!(psi_container::PSIContainer,
             psi_container,
             ts_data_active,
             constraint_name(ACTIVE, L),
-            UpdateRef{L}("get_maxactivepower"),
+            UpdateRef{L}(REAL_POWER, "get_maxactivepower"),  # TODO: reviewers?
```

Comment:
@jd-lara I left lots of these TODO notes. I wasn't sure what the first parameter should be in many of these cases.

Reply:
I will look for them, In this case, it is correct. REAL_POWER and ACTIVE_POWER are used indistinctly. We should improve this and make it consistent.